runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
Your use-case about container and not about process, that's why pre-start hook is ok for you.
I said you my use-cases before - for registering application as working somewhere, so just namespaces and cgroups isn't enough for such task.
I wonder if you think that only your use-cases are true. Pls, believe mine real as well.